tooling/bundler/src/bundle/linux/rpm.rs

```diff
@@ -47,7 +47,8 @@ pub fn bundle_project(settings: &Settings) -> crate::Result<Vec<PathBuf>> {
 
   let license = settings
     .license_file()
```

Comment:
This should be `license`, not `license_file`

Reply:
nice catch